yesod-core-1.6.10

```
Preprocessing test suite 'tests' for yesod-core-1.6.10..
Building test suite 'tests' for yesod-core-1.6.10..
[ 1 of 32] Compiling YesodCoreTest.Auth ( test/YesodCoreTest/Auth.hs, dist/build/tests/tests-tmp/YesodCoreTest/Auth.o )
[ 2 of 32] Compiling YesodCoreTest.Cache ( test/YesodCoreTest/Cache.hs, dist/build/tests/tests-tmp/YesodCoreTest/Cache.o )

test/YesodCoreTest/Cache.hs:69:5: error:
    • No instance for (Control.Monad.Fail.MonadFail (HandlerFor C))
        arising from a do statement
        with the failable pattern ‘Just (V2 v4a)’
    • In a stmt of a 'do' block: Just (V2 v4a) <- cacheByGet "4"
      In the expression:
        do ref <- newIORef 0
           V1 v1a <- cachedBy "1"
                       $ atomicModifyIORef ref $ \ i -> (i + 1, V1 $ i + 1)
           V1 v1b <- cachedBy "1"
                       $ atomicModifyIORef ref $ \ i -> (i + 1, V1 $ i + 1)
           V2 v2a <- cachedBy "1"
                       $ atomicModifyIORef ref $ \ i -> (i + 1, V2 $ i + 1)
           ....
      In an equation for ‘getKeyR’:
          getKeyR
            = do ref <- newIORef 0
                 V1 v1a <- cachedBy "1" $ atomicModifyIORef ref $ \ i -> ...
                 V1 v1b <- cachedBy "1" $ atomicModifyIORef ref $ \ i -> ...
                 ....
   |
69 |     Just (V2 v4a) <- cacheByGet "4"
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Going to add to expected-test-failures for now. CC @snoyberg

--- a/build-constraints.yaml
+++ b/build-constraints.yaml
@@ -4527,7 +4527,6 @@ expected-test-failures:
     - yeshql # https://bitbucket.org/tdammers/yeshql/issues/1/stackage-nightly-test-failure
     - yeshql-core # https://github.com/tdammers/yeshql/issues/6
     - yeshql-hdbc # https://github.com/tdammers/yeshql/issues/6
-    - yesod-core # https://github.com/commercialhaskell/stackage/issues/4319
 
     # Stackage upper bounds, re-enable these when their upper bound is removed
 

